sdks/java/core/src/test/java/org/apache/beam/sdk/io/AvroIOTest.java

```diff
@@ -261,7 +262,9 @@ public void testWriteDisplayData() {
 
     private enum WriteMethod {
       AVROIO_WRITE,
-      AVROIO_SINK
+      AVROIO_SINK,
```

Comment:
`AVROIO_SINK_WITH_SCHEMA`?

Reply:
Ooof -- this is used in different places to distinguish between `write` and `sink` techniques (with and without Schema). 

I think I'll add one more to distinguish between the three types of sinks: **AVROIO_SINK_CLASS** with a specific/generated class, **AVROIO_SINK_WITH_SCHEMA** and **AVROIO_SINK_WITH_FORMATTER**.  What do you think?